packages/management-console/src/components/Organisms/DomainExplorerColumnPicker/DomainExplorerColumnPicker.tsx

```diff
@@ -193,7 +204,7 @@ const DomainExplorerColumnPicker: React.FC<IOwnProps> = ({
             query: gql`
               ${Query.query}
             `,
-            fetchPolicy: 'no-cache'
+            fetchPolicy: enableCache ? undefined : 'network-only'
```

Comment:
```suggestion
            fetchPolicy: enableCache ? 'cache-first' : 'network-only'
```
'cache-first' is the default anyway, just to make the code clear.